### ISSUE TITLE
Каталоги - Силикон

### ISSUE THREAD

**Issue by RusaDrako** [oknosoft/windowbuilder-parametric#68]
С недавнего времени в Кристаллите начали заказываться дополнительные материалы типа "силикон". Сегодня появился первый такой заказ. Попытки найти guid коды для этой позиции не увенчались успехом, т.к. в запрашиваемом каталоге через GET /prm/cat такой позиции нет.

**Comment by unpete** [comment]
Перезапустил сервис - новые позиции в /prm/cat появились.
Просьба проверить наличие цен. Если автоматически не подтянулись - буду разбираться.

**Comment by RusaDrako** [comment]
Позиции появились - спасибо. По поводу цен, прошу уточнить. Они должны 'подтянуться', когда отправляется/формируется заказ?

**Comment by unpete** [comment]
> когда отправляется/формируется заказ?

Да, цены в строках заказов должны соответствовать ожиданиям Кристаллита - как минимум, быть больше нуля.

**Comment by RusaDrako** [comment]
Цена получена. >0

**Comment by unpete** [comment]
@RusaDrako, если нет замечаний, задачу закрываем?
